### PR TITLE
Opensora v1.1 Update

### PR DESCRIPTION
**Pull request by wtomin**
Opensora v1.1:
- [x] vae inference;
- [x] Lattet2v training； 17x512x512, 65x512x512

### CODE REVIEW

File: examples/opensora_pku/README.md [hunk 1]
```diff
@@ -342,30 +313,22 @@ python tools/ckpt/combine_ckpt.py --src output_dir/ckpt --dest output_dir/ckpt -
...
 
 | Model           | Context        | Precision | BS | NPUs | num_frames + num_images| Resolution  | Train T. (s/step) |
 |:----------------|:---------------|:----------|:--:|:----:|:-----------:|:-----------:|:--------------:|
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 4  |  8   |   17 + 4    | 256x256     |   1.8     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 4  |  8   |   65 + 4    | 256x256     |   4.5     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 2  |  8   |   17 + 4    | 512x512     |   3.6     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 4  |  8   |   17 + 4    | 512x512     |   7.5     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 2  |  8   |   65 + 16   | 512x512     |   16.8    |
+| LatteT2V-XL/122 | D910\*x1-MS2.3 | BF16      | 2  |  8   |   17 + 4    | 512x512     |       |
+| LatteT2V-XL/122 | D910\*x1-MS2.3 | BF16      | 2  |  8   |   65 + 16   | 512x512     |       |
```
Comment (SamitHuang) [inline examples/opensora_pku/README.md:331]: MS2.3 master(0615daily)

Comment (SamitHuang) [reply examples/opensora_pku/README.md:331]: https://repo.mindspore.cn/mindspore/mindspore/version/202406/20240615/

Better to add this hyperlink

Comment (SamitHuang) [reply examples/opensora_pku/README.md]: CANN C18 (0517)

Comment (wtomin) [reply examples/opensora_pku/README.md]: Done

**Review by SamitHuang — APPROVED** [review]

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**